packages/mux-player/src/index.ts

```diff
@@ -674,6 +680,55 @@ class MuxPlayerElement extends VideoApiElement {
     this.setAttribute(PlayerAttributes.SECONDARY_COLOR, `${val}`);
   }
 
+  get defaultShowRemainingTime() {
+    return this.hasAttribute(PlayerAttributes.DEFAULT_SHOW_REMAINING_TIME);
+  }
+
+  set defaultShowRemainingTime(val: boolean | undefined) {
```

Comment:
might feel as a stretch but here also I feel this could be part of the controlsList API.

it would require what @gkatsev brought up to have a part that is hidden by default `remainingtime` and the controlsList allowlist could make it show. e.g. `controlslist="noduration remainingtime"`

in Media Chrome these should also be able to be shown / hidden by a CSS var or part I think how they explain in the web component guidelines

Reply:
I'd lean away from using this one with the controls list, not just because it's different from the others with the block vs. allow syntax, but also because clicking on the element toggles between the two display versions.